bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeper.java

```diff
@@ -960,7 +961,8 @@ public void asyncCreateLedgerAdv(final int ensSize, final int writeQuorumSize, f
                 return;
             }
             new LedgerCreateOp(BookKeeper.this, ensSize, writeQuorumSize,
-                               ackQuorumSize, digestType, passwd, cb, ctx, customMetadata).initiateAdv((long)(-1));
+                               ackQuorumSize, digestType, passwd, cb, ctx, customMetadata, LedgerType.PD_JOURNAL)
+                .initiateAdv((long)(-1));
```

Comment:
why did we use cast here? can we change it to `-1L`?

Reply:
define an INVALID_LEDGER_ID somewhere. is -1 special, or would any negative work? Better to use something other than -1, because -1 is so common.

Reply:
ok, but this is not really part of this change. I would like not to review all the usages of -1 and find which -1 are used for INVALID_LEDGER_ID
This line has been changed only in order to add LedgerType.PD_JOURNAL

I will change to -1L